www/js/lib/jquery-3.2.1.js

```diff
@@ -0,0 +1,10253 @@
+/*!
+ * jQuery JavaScript Library v3.2.1
```

Comment:
If we confirm the use of the slim version, we should remove this version (but it was a good idea to keep it for testing the PR before deciding)